laws/shared/src/test/scala/algebra/laws/LawTests.scala

```diff
@@ -143,5 +143,32 @@ trait LawTestsBase extends FunSuite with Discipline {
     implicit val longSemilattice: Semilattice[Long] = LongMinMaxLattice.joinSemilattice
     laws[LatticeLaws, (Set[Int], Long)].check(_.semilattice(lexicographicSemilattice))
   }
+
+  {
+    // This "arbitrary int order" isn't that arbitrary.
+    // I think this could be improved once we have a version of Scalacheck with Cogen.
+    implicit val arbOrderInt: Arbitrary[Order[Int]] = Arbitrary(
+      Gen.oneOf(
+        Gen.const(Order[Int]),
+        Gen.const(Order[Int].reverse)))
+
+    // This is a hack to fake an `Eq` instance for an `Order`.
```

Comment:
Thanks @tixxit. I do think that comparing the sorting results is a better approach. Is the motivation for `N100` to have a smaller domain so that equal elements are more likely?